components/MigrateLocalStorage.coffee

```diff
@@ -2,52 +2,53 @@ noflo = require 'noflo'
 
 # @runtime noflo-browser
 
-class MigrateLocalStorage extends noflo.Component
-  constructor: ->
-    @inPorts =
-      graphstore: new noflo.Port 'object'
+exports.getComponent = ->
+  c = new noflo.Component
+  c.inPorts.add 'graphstore',
+    datatype: 'object'
 
-    @inPorts.graphstore.on 'data', (store) =>
-      @migrateGraphs store
-
-  getGraphs: ->
+  getGraphs = ->
```

Comment:
If these don't rely on the component instance, then better to move them outside to module scope?

Reply:
Fixed in ff410a9